### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by parakh08** [czk32611/GEDDnet#3]
Can you please tell where can I find the processed dataset to run this code?

**Comment by czk32611** [comment]
The normalized data of NISLGaze can be find http://nislgaze.ust.hk/. However, you would need to crop the eyes images by yourself. For the other datasets, please refer to their homepage. I would try to provide the code for preprocessing later.

**Comment by parakh08** [comment]
Okay sure. 
Also, could you just let me know the shape of values in 'face_img' key of 1train.mat (mat['face_img'].shape), that you were using to run your codebase?
Thanks

**Comment by czk32611** [comment]
Please refer to the latest README. The shape of dataset['face_img'] should be N*120*120.

**Comment by parakh08** [comment]
Thanks but I am facing the major problem with the total length of dataset. 

`for ii in range(eye_train.shape[0] // 3000):` part of code in train.py, line 160.

Further, I reduced the constant to 100 because my dataset size is less, but I am facing issue(out of bounds error) on line 61 of tf_utils.py which is `labels_one_hot.flat[index] = 1`.

Thus, I wanted to know the size of the dataset you are using, i.e. `dataset['face_img'].shape[0]`. Could you please help me with that?

**Comment by czk32611** [comment]
Thanks for notifying me this issue. Actually the problem you are facing is that the variable `num_subj` does not metach the data. I have updated to code so that you could input the number of subjects. Note that now you should prepare `dataset['subject_index']`, a column of integer that indexes each subject. Please refer to the latest README.

**Comment by parakh08** [comment]
Thanks for that, although a few things still remain which you must update:
Line 108 in train.py

    `g_hat, t_hat, bias_W_fc, l2_loss = GEDDnet(face, left_eye, right_eye,
                                               keep_prob, is_train, subj_id,
                                               vgg_path=FLAGS.vgg_dir)`
should be updated to 
 
    `g_hat, t_hat, bias_W_fc, l2_loss = GEDDnet(face, left_eye, right_eye,
                                               keep_prob, is_train, subj_id,
                                               vgg_path=FLAGS.vgg_dir, num_subj=num_subj)`

and the testing accuracy part has to be made suitable for the length of dataset.

Further, I also wanted to know how the indexing of 'subject_index' should be, like 0,1,2,... or 1,2,3,...?

**Comment by czk32611** [comment]
Thanks! I will update the code this weekend. The subject index should start at 0. When the image is horizontally flipped, the index should become `i + num_subj` from `i`.

**Comment by parakh08** [comment]
Hello, will it be possible for you to provide me with the full image dataset instead of the normalised images?
The dataset at http://nislgaze.ust.hk/original_data/p01.tar.gz is not accessible to me.

**Comment by javierBarandiaran** [comment]
Is the dataset still available somewhere?
